src/project/nativeScriptCli.ts

```diff
@@ -60,26 +19,18 @@ export class NativeScriptCli {
         if (utils.getPlatform() === utils.Platform.Windows) {
             this._shellPath = "cmd.exe";
         }
+    }
+
+    public get path(): string { return this._path; }
 
-        let versionStr = null;
+    public executeGetVersion(): string {
         try {
-            versionStr = this.executeSync(["--version"], undefined);
+            return this.executeSync(["--version"], undefined);
```

Comment:
On macOS Sierra and below, this line will return:
```
Support for macOS 10.12 is deprecated and will be removed in one of the next releases of NativeScript. Please, upgrade to the latest macOS version.
4.1.2
```

This is not a valid semver version and using semver with it will fail. Consider using [such approach](https://github.com/NativeScript/nativescript-doctor/blob/master/lib/sys-info.ts#L429-L432)